### PR TITLE
Fixed 403 Response error and updated modules

### PR DESCRIPTION
**Pull request by jmrchelani**
Fixed issue: [HTTP error 403 #18](https://github.com/devcreatives/md2urdu/issues/18)
Also updated node modules

### CODE REVIEW

**Comment by jmrchelani** [comment]
There are problems with test or smth I couldnt directly push from Github Desktop so check that out. Otherwise Everything is perfectly working. I have uploaded the vsix file as well for checking.

File: src/lib/extension.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 // The module 'vscode' contains the VS Code extensibility API
 // Import the module and reference it with the alias vscode in your code below
 import * as vscode from 'vscode';
-import fs from 'fs';
-import translate from '@vitalets/google-translate-api';
+const fs = require('fs');
```
Comment (MrSlimCoder) [inline src/lib/extension.ts:4]: Why require instead of import?

File: package.json [hunk 2]
```diff
@@ -1,59 +1,59 @@
...
+		"test:jest": "jest",
+		"prettier": "prettier --write src/**/*.ts"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "npm run prettier",
+			"pre-push": "npm run test:jest"
+		}
+	},
+	"devDependencies": {
```
Comment (MrSlimCoder) [inline package.json:40]: Why update all dependencies?

**Comment by MrSlimCoder** [comment]
@jmrchelani I think you are using some beauitfy type of extension that's why it changed stylijg of codebase?

**Review by MrSlimCoder — APPROVED** [review]

**Comment by MrSlimCoder** [comment]
Merging this PR.

**Comment by MrSlimCoder** [comment]
@allcontributors please add @jmrchelani for code.

**Pull request merged**